lib/const.js

```diff
@@ -48,61 +48,61 @@ const comfoCommands = [
 ];
 
 const sensorCodes = [
-  {code:  16, kind: 1, name: ''},
+  {code:  16, kind: 1, name: 'SENSOR_AWAY_INDICATOR'}, // (01 = low, medium, high fan speed, 07 = away)
```

Comment:
for clarity: since all transmitted data are hexadecimal, add '0x' before or 'h' after.
That way, it's much clearer for other users.
This comment is valid for all other changes in this file :-)

Reply:
I've copied the comments from the documentation of pycomfoconnect